.github/workflows/import-to-elasticsearch.yml

```diff
@@ -58,6 +53,11 @@ jobs:
       with:
         CACHIX_SIGNING_KEY: ${{ secrets.CACHIX_SIGNING_KEY }}
 
+    - name: Installing Nix unstable
+      run: |
+        nix-env -f '<nixpkgs>' -iA nixVersions.unstable # for https://github.com/NixOS/nix/pull/6242
+        echo "$HOME/.nix-profile/bin" >> "$GITHUB_PATH"
+
```

Comment:
Why not add this to the `setup` action?
You can provision specific nix versions with `cachix/setupNix`

Reply:
It's really only needed for the nixpkgs import so there's no need to fetch nixUnstable in every other job.

I know you can specify `install_url` when installing Nix but it seems clearer this way

Reply:
Ok fair enough